src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -287,11 +287,9 @@ export default class ArrayField extends React.Component {
 
       if (wrapper) {
         const focusableElements = getFocusableElements(wrapper);
-        const firstFocusableElement = wrapper.querySelector(
-          focusableElements[0].tagName,
-        );
-
-        firstFocusableElement.focus();
+        if (focusableElements.length) {
```

Comment:
I couldn't think of any good reasoning why tagName would be targeted here explicitly. It seems like we would just want to focus the first element. and tagName doesn't always play nice with web components.